lib/server.js

```diff
@@ -135,7 +129,10 @@ export default function main() {
                 workerPid: worker.process.pid,
             });
             setTimeout(() => {
-                if (!worker.isDead()) {
+                // TODO: When upgrading to Node v6,
+                // instead of !worker.suicide,
+                // use !worker.exitedAfterDisconnect
```

Comment:
Why using `!worker.exitedAfterDisconnect` in node v6? There is still IPC issue: http://ci.ironmann.io/gh/scality/Integration/10757.